docs/capacity-planning.rst

```diff
@@ -0,0 +1,326 @@
+.. _ksql_capacity_planning:
+
+======================
+KSQL Capacity Planning
+======================
+
+.. contents::
+
+KSQL is a simple and powerful tool for building streaming applications on top of Kafka. This guide will help you get started with planning for provisioning your KSQL deployment by helping you answer questions like:
+
+    - What server specification should I use to run KSQL?
+    - Roughly how many KSQL server nodes will I need?
+    - Do I need to provision additional Kafka brokers to support KSQL?
+    - What kind of deployment mode makes sense for me?
+    - How do I know whether my KSQL queries are handling the incoming message rate, and how can I tune KSQL if it’s falling behind?
+
+Background
+==========
+
```

Comment:
suggest:

```
.. tip:: Because the underlying implementation of KSQL uses the `Kafka Streams API
         <https://docs.confluent.io/current/streams/developer-guide/dsl-api.html>`_  for query
         processing, the details covered in the Streams documentation apply to KSQL as well. The `Kafka
         Streams capacity planning guide <https://docs.confluent.io/current/streams/sizing.html>`_
         is another useful resource for KSQL capacity planning.
```